src/org/usfirst/frc/team4915/steamworks/commands/DriveStraightCommand.java

```diff
@@ -20,7 +20,7 @@
     private static final double m_KP = .1875;
     private static final double MAX_DEGREES_ERROR = .125;
 
-    private static final double k_P = 1.7, k_D = 0, k_I = 0, k_F = 0;
+    private static final double k_P = 1.7, k_D = 0, k_I = 0, k_F = 0; // P was 1.7
```

Comment:
This is in case I make any changes in the future, it's so I don't have to go back to the diff.